apps/web/components/PoolGovernance.tsx

```diff
@@ -98,12 +100,19 @@ export const PoolGovernance = ({
               )}
             </div>
           </div>
-          <ActivatePoints
-            strategyAddress={strategy.id as Address}
-            communityAddress={communityAddress}
-            isMemberActivated={isMemberActivated as boolean | undefined}
-            isMember={isMember}
-          />
+          <div className="flex flex-col gap-2">
+            <CheckPassport
+              strategyAddr={strategy.id as Address}
+              enableCheck={!isMemberActivated}
+            >
+              <ActivatePoints
+                strategyAddress={strategy.id as Address}
+                communityAddress={communityAddress}
+                isMemberActivated={isMemberActivated as boolean | undefined}
```

Comment:
![Screenshot from 2024-07-18 18-19-19](https://github.com/user-attachments/assets/7c54dff3-50ee-48a2-b86e-18ef24e7364d)

Reply:
actually i do not know how to tell typescript that this strins is actually a `0x${string}` when it comes from the subgraph...

Reply:
Sorry I was talking about the as boolean, yeah as Address we sometimes dont have choice

Reply:
> actually i do not know how to tell typescript that this strins is actually a `0x${string}` when it comes from the subgraph...

Problem is type Id of subgraph is not always starting with 0x it can be any string so they wont map the types to reflect that. But yeah thats ok we can have the as Address exception.